lib/exercises.js

```diff
@@ -1,17 +1,61 @@
 // This method will return an array of arrays.
 // Each subarray will have strings which are anagrams of each other
-// Time Complexity: ?
-// Space Complexity: ?
+// Time Complexity: O(n)^2
+// Space Complexity: O(n)^2
 function grouped_anagrams(strings) {
```

Comment:
👍  I would say this is O(m * n log n) for time complexity where m is the number of strings and n is the avg length of each string.